akka-stream/src/main/scala/akka/stream/Fusing.scala

```diff
@@ -34,6 +34,12 @@ object Fusing {
   def aggressive[S <: Shape, M](g: Graph[S, M]): FusedGraph[S, M] = Impl.aggressive(g)
 
   /**
+   * Return the StructuralInfo for this Graph without any fusing
+   */
+  def structuralInfo[S <: Shape, M](g: Graph[S, M], attributes: Attributes): StructuralInfoModule =
```

Comment:
Thanks @patriknw - I added the necessary filter in MiMa.scala for the internal type StructuralInfo which is now StructuralInfoModule. Hopefully this build will now pass.